### PR TITLE
media-sound/fluidsynth: Fix implicit declaration of function strcasecmp

### PR DESCRIPTION
**Pull request by listout**
Closes: https://bugs.gentoo.org/906150

### CODE REVIEW

File: media-sound/fluidsynth/files/2.3.2-musl-implicit-declaration-fix.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
+From 8964c64644349c7f232234b51423b2ec5e7c894e Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline media-sound/fluidsynth/files/2.3.2-musl-implicit-declaration-fix.patch:1]: No upstream link?

Comment (listout) [reply media-sound/fluidsynth/files/2.3.2-musl-implicit-declaration-fix.patch]: Sorry, I didn't submit the patch upstream back then. Created upstream PR, linked in the patch and ran scrub-patch on the file.

Comment (listout) [reply media-sound/fluidsynth/files/2.3.2-musl-implicit-declaration-fix.patch]: Do I need all the PR link, currently I've added the patch-diff (thinking of preservation, although github PR's are preserved I think)

Comment (thesamesam) [reply media-sound/fluidsynth/files/2.3.2-musl-implicit-declaration-fix.patch]: I normally include the "regular" PR link instead so it's easier to see the discussion but this is fine too, thanks

File: media-sound/fluidsynth/files/2.3.2-musl-implicit-declaration-fix.patch [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Subject: [PATCH] Fix implicit declaration of function strcasecmp
+
+Signed-off-by: Brahmajit Das <brahmajit.xyz@gmail.com>
+Bug: https://bugs.gentoo.org/906150
+--- a/src/utils/fluidsynth_priv.h
++++ b/src/utils/fluidsynth_priv.h
+@@ -43,6 +43,7 @@
+
+ #if HAVE_STRING_H
+ #include <string.h>
```
Comment (thesamesam) [inline media-sound/fluidsynth/files/2.3.2-musl-implicit-declaration-fix.patch:13]: Actually, is there a HAVE_STRINGS_H? It should go under that (might need to add it to configure)

Comment (listout) [reply media-sound/fluidsynth/files/2.3.2-musl-implicit-declaration-fix.patch:13]: On it.

Comment (listout) [reply media-sound/fluidsynth/files/2.3.2-musl-implicit-declaration-fix.patch:23]: How about now?

**Pull request closed without merging**